src/number_theory/legendre_symbol/quadratic_char.lean

```diff
@@ -67,7 +67,7 @@ begin
 end
 
 @[simp]
-lemma quadratic_char_zero : quadratic_char_fun F 0 = 0 :=
+lemma quadratic_char_zero' : quadratic_char_fun F 0 = 0 :=
```

Comment:
```suggestion
lemma quadratic_char_fun_zero : quadratic_char_fun F 0 = 0 :=
```
I'm suggesting a slightly longer name, but this way it will be immediately clear to users what this variant is doing. Which is helpful, e.g. in auto-complete.

Reply:
OK. I'm also renaming the other lemmas referring to `quadratic_char_fun`.